### DISCUSSION TITLE
SOAP requests started timing out recently

### DISCUSSION THREAD

**Discussion by elidrissidev** [OpenMage/magento-lts#2223] [Questions & Answers]
It's been a few weeks now, I started noticing that SOAP requests to Magento were timing out very often with this error:
```
ERROR - 2022-06-14 09:43:11 --> Severity: Warning --> SoapClient::SoapClient(): SSL: Connection reset by peer /path/to/file/that/performs/soap/request 7
ERROR - 2022-06-14 09:43:11 --> Severity: Warning --> SoapClient::SoapClient(http://schemas.xmlsoap.org/soap/encoding/): failed to open stream: HTTP request failed! /path/to/file/that/performs/soap/request 7
ERROR - 2022-06-14 09:43:11 --> Severity: Warning --> SoapClient::SoapClient(): I/O warning : failed to load external entity &quot;http://schemas.xmlsoap.org/soap/encoding/&quot; /path/to/file/that/performs/soap/request 7
```

When I try to access https://schemas.xmlsoap.org/soap/encoding/ from the browser, sometimes it also times out with similar errors 🙃. Anyone facing a similar issue?

<img width="861" alt="Screen Shot 2022-06-14 at 9 57 17 AM" src="https://user-images.githubusercontent.com/67818913/173537448-0578e053-cba1-4cc8-9595-71a3fa40aef3.png">

**Comment by fballiano** [comment]
I think Microsoft lowered the request limit cause it was happening to one of my customer too, we had to setup a fake mirror of the schemas to avoid calling the upstream servers...

**Reply by elidrissidev** [reply]
And how did you configure Magento to use it? You modified wsdl files?

**Reply by fballiano** [reply]
modifying the /etc/hosts of the servers

it's kinda impossible to modify the wsdl files because some of them are kinda "recursive", they're downloaded at runtime so we'd have no control on those (unless doing this work recursively)
